go_node_engine/cmd/conf.go

```diff
@@ -23,9 +23,13 @@ func init() {
 	setAuth.Flags().StringVarP(&certFile, "certFile", "c", "", "Path to certificate for TLS support")
 	setAuth.Flags().StringVarP(&keyFile, "keyFile", "k", "", "Path to key for TLS support")
 	setVirtualizationCmd.AddCommand(enableUnikernel)
+	setCni.AddCommand(explainNetManager)
 	setCni.AddCommand(enableNetwork)
 	setCni.AddCommand(disableNetwork)
 	setCni.AddCommand(enableManualNetwork)
+	setCni.AddCommand(explainIP)
+	setCni.AddCommand(publicIP)
+	setCni.AddCommand(privateIP)
```

Comment:
Maybe I misunderstood, shouldn't this be part of explainIP command?
Like:
```suggestion
	explainIP.AddCommand(publicIP)
	explainIP.AddCommand(privateIP)
```

Reply:
Originally I was trying to shoehorn public/private IP networking into the network command. But I think adding a new command called 'visibility' is cleaner. What do you think?